src/platforms/common/index.mdx

```diff
@@ -0,0 +1,62 @@
+On this page, we get you up and running with Sentry's SDK, so that it will automatically report errors and exceptions in your application.
+
+<Note>
+<markdown>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</markdown>
+</Note>
+
+<PlatformContent includePath="framework-list" />
```

Comment:
btw one thing you might do, is put a _default.mdx in the framework-list, and then in that just put `<GuideGrid />`.

caveat: it will error on child pages right now.. so i need to fix that (going to push a fix into your branch)

Reply:
thanks. I will do that.